docs/build/testing-framework.md

```diff
@@ -0,0 +1,183 @@
+# Testing Framework Documentation
+
+## Overview
+
+This document outlines the testing framework used for Subquery projects. The testing framework provides an easy way to test the behavior of mapping handlers and validate the data being indexed.
+
+## Usage
+
+To use the testing framework, you need to:
+
+1. Create test files with the naming convention *.test.ts and place them in the src/tests folder. Each test file should contain test cases for specific mapping handlers.
```

Comment:
Should we include the build step here?

Reply:
It won't hurt. What are they? yarn codegen, & build?